Library/Homebrew/cmd/update.sh

```diff
@@ -242,6 +242,14 @@ EOS
   INITIAL_BRANCH="$(git symbolic-ref --short HEAD 2>/dev/null)"
   if [[ "$INITIAL_BRANCH" != "$UPSTREAM_BRANCH" && -n "$INITIAL_BRANCH" ]]
   then
+
+    if [[ -z "$HOMEBREW_DEVELOPER" ]]
+    then
+      echo "Checking out $UPSTREAM_BRANCH in $DIR..."
+      echo "To checkout $INITIAL_BRANCH in $DIR run:"
+      echo "  'cd $DIR && git checkout $INITIAL_BRANCH"
```

Comment:
Not quoting in `echo` is fine, I think, but thanks for the review anyway @zmwangx; definitely would rather have pedantry than the other extreme!